relay-server/src/actors/global_config.rs

```diff
@@ -0,0 +1,153 @@
+use std::sync::Arc;
+use std::time::Duration;
+
+use relay_dynamic_config::GlobalConfig;
+use relay_system::{Addr, AsyncResponse, FromMessage, Interface, Sender, Service};
+use tokio::sync::watch;
+
+use crate::actors::project_upstream::{GetProjectStates, GetProjectStatesResponse};
+use crate::actors::upstream::{SendQuery, UpstreamRelay, UpstreamRequestError};
+
+/// Service implementing the [`GlobalConfigInterface`] interface.
+///
+/// The service offers two alternatives to fetch the [`GlobalConfig`]:
+/// responding to a [`GlobalConfigInterface::Get`] message with the config for
+/// one-off requests, or subscribing to updates with
+/// [`GlobalConfigInterface::Subscribe`] to keep up-to-date.
```

Comment:
Better link to the `Subscribe` type here, since that is what callers will use. They invoke this as `global_config.send(Subscribe)`.